src/main/java/org/jabref/logic/cleanup/ConvertToBiblatexCleanup.java

```diff
@@ -39,8 +39,18 @@
                 entry.clearField(StandardField.YEAR).ifPresent(changes::add);
                 entry.clearField(StandardField.MONTH).ifPresent(changes::add);
             });
+        } else {
+            // If date field is filled and is equal to year it should be removed the year and month fields
+            entry.getFieldOrAlias(StandardField.DATE).ifPresent(date -> {
+                entry.getFieldOrAlias(StandardField.YEAR).ifPresent(yearl -> {
```

Comment:
Can you give the variable `year1` a more meaningful name? Sure, everybody understands that, but nevertheless, we try to write exemplary code in JabRef (I know, it's still a loooooong way to go).

Reply:
you're right, I forgot to change

Reply:
Done.

Reply:
Done.